pkg-manager/plugin-commands-installation/src/sync.ts

```diff
@@ -0,0 +1,97 @@
+import { docsUrl } from '@pnpm/cli-utils'
```

Comment:
@zkochan hello! It's my first time contributing anything other than an error message to pnpm, so if anything looks super wonkey / wrong -- lemme know -- I haven't written tests yet, and the PR is still marked as draft, but I wanted to ping you for early review so that you may have a chance to look at this before I end up spending too much time going down a wrong direction.

This file is < 100 lines :shrug: if that helps prioritization (this is a very busy project!)